### ISSUE TITLE
[win10_minGW] cannot create moodbar file under network connection (Samba SMB)

### ISSUE THREAD

**Issue by Calio-Templar** [strawberrymusicplayer/strawberry#1101]
- [v ] I have checked the [FAQ](https://wiki.strawberrymusicplayer.org/wiki/FAQ) for answers.
- [v ] I have checked the [Changelog](https://github.com/strawberrymusicplayer/strawberry/blob/master/Changelog) that the issue is not already fixed. 
- [v ] I believe this issue is a bug, and not a general technical issue, question or feature requests that can be discussed on the [forum](https://forum.strawberrymusicplayer.org/). 

**Describe the bug**
Playing network files from raspberry pi SMB server fails to produce moodbar data and .mood file

**To Reproduce**
Steps to reproduce the behavior.
1. Access some non-local files, ideally Samba server files through windows network share, and add file to music list.
2. File can be played but no mood file generates, as from settings to save *.mood file in the songs folders

**Expected behavior**
*.mood file should be produced and fancy colorful progress bar should be made.

**Screenshots:**
debug output:
01:08:32.035 DEBUG Windows7ThumbBar:179             Updating 6 buttons
01:08:34.076 DEBUG PlaylistBackend:330              Saving playlist 1
01:08:42.712 DEBUG SongLoader:231                   Loading local file "//caliopi/pi/Music/Retribution.mp3"
01:08:42.723 DEBUG _MessageReplyBase:31             Waiting on ID 597
01:08:42.723 DEBUG TagReaderTagLib:151              Checking for valid file "//caliopi/pi/Music/Retribution.mp3"
01:08:42.733 DEBUG MessageReply<MessageType>:90     Releasing ID 597 (finished)
01:08:42.736 DEBUG _MessageReplyBase:33             Acquired ID 597
01:08:42.741 DEBUG _MessageReplyBase:31             Waiting on ID 598
01:08:42.742 DEBUG TagReaderTagLib:194              Reading tags from "//caliopi/pi/Music/Retribution.mp3"
01:08:42.756 DEBUG MessageReply<MessageType>:90     Releasing ID 598 (finished)
01:08:42.761 DEBUG _MessageReplyBase:33             Acquired ID 598
01:08:42.768 DEBUG Playlist:1175                    Updating playlist with new tracks' info
01:08:42.778 INFO  MoodbarLoader:162                Creating moodbar data for "//caliopi/pi/Music/Retribution.mp3"
01:08:42.785 ERROR MoodbarPipeline:131              Error processing QUrl("file://caliopi/pi/Music/Retribution.mp3") : "Resource not found."
01:08:42.796 ERROR MoodbarPipeline:131              Error processing QUrl("file://caliopi/pi/Music/Retribution.mp3") : "Could not open resource for reading."
01:08:42.804 ERROR MoodbarPipeline:131              Error processing QUrl("file://caliopi/pi/Music/Retribution.mp3") : "GStreamer error: state change failed and some element failed to post a proper error message with the reason for the failure."
01:08:42.817 ERROR MoodbarPipeline:131              Error processing QUrl("file://caliopi/pi/Music/Retribution.mp3") : "Resource not found."
01:08:42.825 ERROR MoodbarPipeline:131              Error processing QUrl("file://caliopi/pi/Music/Retribution.mp3") : "Could not open resource for reading."
01:08:42.831 ERROR MoodbarPipeline:131              Error processing QUrl("file://caliopi/pi/Music/Retribution.mp3") : "GStreamer error: state change failed and some element failed to post a proper error message with the reason for the failure."
01:08:43.675 DEBUG PlaylistBackend:330              Saving playlist 1

**System Information:**
 - Operating system: Windows 10 22h2
 - Strawberry Version: 1.0.13 minGW. MSVC version crashes.

**Comment by jonaski** [comment]
Is it the 64 bit MSVC version that is crashing?
Could you try uninstalling Visual C++ 2015-2022 Redistributable from the control panel, and reinstall it from here: https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170
If that does not work, could you open a separate issue for that and paste the info about the crash from the Windows Application log.
